src/logic/Board.ts

```diff
@@ -7,26 +7,29 @@ import {
   ring,
   spiral,
   toCube,
+  Traverser,
 } from "honeycomb-grid";
 
 import IBoard from "../types/IBoard";
-import ITile from "../types/ITile";
 import { Tile } from "./Tile";
 
 export class Board implements IBoard {
-  grid: Grid<ITile>;
+  grid: Grid<Tile>;
```

Comment:
why does this use Tile rather than ITile?

Reply:
ITile also works, but using Tile guarantees that the grid only contains instances of the Tile class (if there's an object that implements ITile but isn't an instance of the Tile class, we might face runtime issues). ITile would allow the grid to contain any object that adheres to the ITile interface, but I don't think we need this flexibility. So I think the convention would be to use Tile.